changelogs/unreleased/gh-8418-asserton-on-select-from-_space.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/sql
+
+* Fixed assertion when selecting from '_space' using 'owner' field in
+  WHERE (gh-8418).
```

Comment:
```suggestion
* Fixed an assertion when selecting from `_space` using the `owner` field in
  `WHERE` (gh-8418).
```

Reply:
Thank you for the review! I changed changelog a bit.